### PR TITLE
[KO] Translation of keras_model_to_estimator.ipynb

### PR DESCRIPTION
**Pull request by jaketae**
Uploaded translation of tutorials/estimator/keras_model_to_estimator.ipynb

### CODE REVIEW

File: site/ko/tutorials/estimator/keras_model_to_estimator.ipynb [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+      },
+      "outputs": [],
+      "source": [
+        "try:\n",
+        "  # %tensorflow_version 명령은 코랩에서만 쓸 수 있습니다.\n",
+        "  %tensorflow_version 2.x\n",
+        "except Exception:\n",
+        "  pass"
+      ]
+    },
```
Comment (rickiepark) [inline site/ko/tutorials/estimator/keras_model_to_estimator.ipynb:150]: These cells is removed in en version.

File: site/ko/tutorials/estimator/keras_model_to_estimator.ipynb [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "ABgo9-8BtYNs"
+      },
+      "source": [
+        "모델을 컴파일한 후, 모델 구조의 요약도를 볼 수 있습니다."
```
Comment (rickiepark) [inline site/ko/tutorials/estimator/keras_model_to_estimator.ipynb:213]: '모델 구조의 요약도를 볼' --> '모델 구조를 요약해 출력할'

File: site/ko/tutorials/estimator/keras_model_to_estimator.ipynb [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "pM3Cx5Fm_sHI"
+      },
+      "source": [
+        "### 입력 함수 만들기\n",
+        "\n",
+        "[데이터셋 API](../../guide/data.md)를 사용해 대규모 데이터셋을 확장하거나 다중 기기 학습(multi-device training)을 실행할 수 있습니다.\n",
```
Comment (rickiepark) [inline site/ko/tutorials/estimator/keras_model_to_estimator.ipynb:240]: '데이터셋을 확장하거나' --> '데이터셋을 다루거나'
'다중 기기 학습' --> '여러 장치에서 훈련'

File: site/ko/tutorials/estimator/keras_model_to_estimator.ipynb [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+        "\n",
+        "텐서플로 추정기는 입력 파이프라인(input pipeline)이 언제 어떻게 생성었는지 제어할 수 있어야 합니다. 이를 위해서는 \"입력 함수\", 즉 `input_fn`이 필요합니다. 추정기는 이 함수를 별도의 매개변수 설정 없이 호출하게 됩니다. 이때 `input_fn`은 `tf.data.Dataset` 객체를 반환해야 합니다."
```
Comment (rickiepark) [inline site/ko/tutorials/estimator/keras_model_to_estimator.ipynb:242]: '생성었는지' --> '생성되었는지'
'제어할 수 있어야 합니다' --> '제어해야 합니다'

File: site/ko/tutorials/estimator/keras_model_to_estimator.ipynb [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "svdhkQ4Otcv0"
+      },
+      "source": [
+        "### tf.keras.model을 추정기로 변환하기\n",
+        "\n",
+        "`tf.keras.model`은 `tf.keras.estimator.model_to_estimator` 함수를 이용해 `tf.estimator.Estimator` 객체로 변환함으로써 `tf.estimator` API를 통해 학습시킬 수 있습니다."
```
Comment (rickiepark) [inline site/ko/tutorials/estimator/keras_model_to_estimator.ipynb:297]: '학습시킬' --> '훈련할'

File: site/ko/tutorials/estimator/keras_model_to_estimator.ipynb [hunk 6]
```diff
@@ -0,0 +1,343 @@
...
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "U-8ekW5It_2w"
+      },
+      "source": [
+        "추정기를 학습시킨 후 평가합니다."
```
Comment (rickiepark) [inline site/ko/tutorials/estimator/keras_model_to_estimator.ipynb:324]: '학습시킨' --> '훈련한'

File: site/ko/tutorials/estimator/keras_model_to_estimator.ipynb [hunk 7]
```diff
@@ -0,0 +1,343 @@
...
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "ouIkVtp9uAg5"
+      },
+      "outputs": [],
+      "source": [
+        "keras_estimator.train(input_fn=input_fn, steps=500)\n",
+        "eval_result = keras_estimator.evaluate(input_fn=input_fn, steps=10)\n",
+        "print('Eval result: {}'.format(eval_result))"
```
Comment (rickiepark) [inline site/ko/tutorials/estimator/keras_model_to_estimator.ipynb:339]: Please translate 'Eval result' :)

**Comment by jaketae** [comment]
@rickiepark, thank you once again for the detailed review and comments. I've applied your suggestions. Please take a look when you have time and let me know if there's anything else I can do to improve the translation!

**Comment by rickiepark** [comment]
Thank you @jaketae 
LGTM 👍 👍 👍

**Pull request merged**